Src/main.c

```diff
@@ -70,6 +70,12 @@ void SystemClock_Config(void);
 int main(void)
 {
 
+  uint32_t ADCValue = 0;
+  // Period * freq / 1000ms = cycles/ms
+  // 60000 * 50 / 1000 = 3000
+  static const uint16_t Pulse_Range = 3000; //1ms max pulse on-time
+  static const uint16_t Pulse_Delay = 3000; //1ms pulse start time
+  static const uint16_t ADC_Resolution = 4095; //ADC has 12bit resolution
```

Comment:
Don't need 'static' keyword within the main method

The keyword is really used in these contexts:
- Creating a parameter in a class that should be the same for all instances of the class
- Creating a global variable that you only want to have file scope